README.md

```diff
@@ -225,6 +225,41 @@ ping_type = "udp"
 
 [HCL]: https://github.com/hashicorp/hcl "HashiCorp Configuration Language (HCL)"
 
+### Consul ACL Policies
+
+With [ACL system][ACL] enabled on Consul agents, a specific ACL policy may be
+required for ESM's token in order for ESM to perform its functions. To narrow
+down the privileges required for ESM the following [ACL policy rules][rules]
+can be used:
+
+```hcl
+agent_prefix "" {
+  policy = "write"
```

Comment:
```suggestion
  policy = "read"
```

Reply:
I only found one API that uses agent policies: `/agent/self` used [here](https://github.com/hashicorp/consul-esm/blob/ef82992b1097d1a59f27bcc96f152e8de7bdb7a3/coordinate.go#L336) and [here](https://github.com/hashicorp/consul-esm/blob/00a22948c3ced16e08a3dafa3d080281e161791f/agent.go#L444
). It only requires [`agent:read`](https://www.consul.io/api-docs/agent#read-configuration)